scripts/spack/packages/mfem/package.py

```diff
@@ -0,0 +1,664 @@
+import os
+
+from spack.package import *
+from spack.pkg.builtin.mfem import Mfem as BuiltinMfem
+
+
+class Mfem(BuiltinMfem):
+
+    version(
+        "4.8.0",
+        sha256="49bd2a076b0d87863092cb55f8524b5292d9afb2e48c19f80222ada367819016",
+        url="https://bit.ly/mfem-4-8",
+        extension="tar.gz",
+    )
+
+    ## Hack (KW,14May2024): We needed to add the `hipblas` library to the hip_libs
+    ## I've copied the entire `get_make_config_options` function
+    ## to inject the appropriate call below.
+    ## We should remove this entire function once we can update 
+    ## to a spack package that contains this change.
+    ## It looks like spack@develop already has a fix for this
+    def get_make_config_options(self, spec, prefix):
```

Comment:
Note: no need to review the bulk of this file. This function is copied from the mfem package in spack, and I added a small patch to the hip libraries, demarcated by comments with `AXOM PATCH`.